cmd/neofs-lens/internal/objects/root.go

```diff
@@ -0,0 +1,21 @@
+package objects
+
+import "github.com/spf13/cobra"
+
+var vPath string
+
+// Root defines root command for operations with NeoFS objects.
+var Root = &cobra.Command{
+	Use:   "objects",
+	Short: "NeoFS objects inspection",
+}
+
+const filePathFlagKey = "file"
+
+func init() {
+	Root.PersistentFlags().StringVar(&vPath, filePathFlagKey, "", "Path to file with raw NeoFS object payload")
+	_ = Root.MarkFlagFilename(filePathFlagKey)
```

Comment:
```suggestion
	_ = Root.MarkPersistentFlagFilename(filePathFlagKey)
```
now u'll get 'flag not found' error (ignored). I see u check this in `linkFunc`, this case is still better to be caught by Cobra engine

same below

if arrived with copypaste, lets leave an issue for other cases (or make instafix). Seems like ignoring such errors is not so good

Reply:
fixed

> if arrived with copypaste, lets leave an issue for other cases

no, done myself but remembered that we ignore this err usually. do you mean we need not to ignore such error ever or just fixing `*Persistent*` here?

Reply:
> first u get fs := Root.PersistentFlags() and then working with fs only

ok, lets try this

> leave an issue for other cases

https://github.com/nspcc-dev/neofs-node/issues/2800